modules/cli/src/main/scala/scala/cli/commands/shared/SuppressWarningOptions.scala

```diff
@@ -0,0 +1,19 @@
+package scala.cli.commands.shared
+
+import caseapp.*
+
+import scala.cli.commands.tags
+
+// format: off
+final case class SuppressWarningOptions(
+  @Tag(tags.implementation)
+  @HelpMessage("Suppress warnings about using directives in multiple files")
+  @Name("suppress-warning-directives-in-multi-files")
+    multiFileDirectivesWarning: Boolean = false
```

Comment:
```suggestion
  @Hidden
  @Name("suppressWarningDirectivesInMultipleFiles")
    suppressDirectivesInMultipleFilesWarning: Boolean = false
```
You can use camel case in `@Name` aliases.
Also, I'm thinking if we could find a shorter alias that would bring this flag's name down to a sensible length... can't think of a better one atm 😐